ucb-rit/coldfront
# Add script to load PostgreSQL dump file into local database

Fixes #256

**Changes**
- Added a script that loads a database dump file generated by `pg_dump` into the local PostgreSQL (9.6) database.

**How to Test**
- Review the code changes and add comments as needed.
- Manual Testing
    - Set up the VM provided by #255.
    - Within the VM, run the script and confirm that data are loaded into the application.
      ```
      sh load_database_backup.sh /path/to/file.dump
      ```

**Notes**
- This pull request should not be merged until after #255 is.
- The method for fetching such a dump file will be described elsewhere.
